Mathlib/LinearAlgebra/Alternating.lean

```diff
@@ -143,30 +148,30 @@ theorem ext {f f' : AlternatingMap R M N ι} (H : ∀ x, f x = f' x) : f = f' :=
 #align alternating_map.ext AlternatingMap.ext
 
 theorem ext_iff {f g : AlternatingMap R M N ι} : f = g ↔ ∀ x, f x = g x :=
-  ⟨fun h x => h ▸ rfl, fun h => ext h⟩
+  ⟨fun h _ => h ▸ rfl, fun h => ext h⟩
 #align alternating_map.ext_iff AlternatingMap.ext_iff
 
-instance : Coe (AlternatingMap R M N ι) (MultilinearMap R (fun i : ι => M) N) :=
+attribute [coe] AlternatingMap.toMultilinearMap
+
+instance coe : Coe (AlternatingMap R M N ι) (MultilinearMap R (fun _ : ι => M) N) :=
   ⟨fun x => x.toMultilinearMap⟩
+#align alternating_map.multilinear_map.has_coe AlternatingMap.coe
 
 @[simp, norm_cast]
-theorem coe_multilinearMap : ⇑(f : MultilinearMap R (fun i : ι => M) N) = f :=
+theorem coe_multilinearMap : ⇑(f : MultilinearMap R (fun _ : ι => M) N) = f :=
   rfl
 #align alternating_map.coe_multilinear_map AlternatingMap.coe_multilinearMap
 
 theorem coe_multilinearMap_injective :
-    Function.Injective (coe : AlternatingMap R M N ι → MultilinearMap R (fun i : ι => M) N) :=
-  fun x y h => ext <| MultilinearMap.congr_fun h
+    Function.Injective ((↑) : AlternatingMap R M N ι → MultilinearMap R (fun _ : ι => M) N) :=
+  fun _ _ h => ext <| MultilinearMap.congr_fun h
 #align alternating_map.coe_multilinear_map_injective AlternatingMap.coe_multilinearMap_injective
 
-@[simp]
-theorem toMultilinearMap_eq_coe : f.toMultilinearMap = f :=
-  rfl
-#align alternating_map.to_multilinear_map_eq_coe AlternatingMap.toMultilinearMap_eq_coe
+#noalign alternating_map.to_multilinear_map_eq_coe
 
 @[simp]
 theorem coe_multilinearMap_mk (f : (ι → M) → N) (h₁ h₂ h₃) :
-    ((⟨f, h₁, h₂, h₃⟩ : AlternatingMap R M N ι) : MultilinearMap R (fun i : ι => M) N) =
+    ((⟨⟨f, h₁, h₂⟩, h₃⟩ : AlternatingMap R M N ι) : MultilinearMap R (fun _ : ι => M) N) =
```

Comment:
This should be changed to
```suggestion
    ((⟨f, h₃⟩ : AlternatingMap R M N ι) : MultilinearMap R (fun _ : ι => M) N) =
```
along with a porting note referring to `mk` changing, and a subscript `x` on the align.